js/index.js

```diff
@@ -40,3 +40,114 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+//===================Navigation bar text:===================//
+const navLinks = document.querySelectorAll('nav a');
+// Array.from(navLinks);
+// for each anchor tag, give it the value listed in the siteContent object
+navLinks.forEach((v, i) => v.textContent = siteContent["nav"][`nav-item-${i+1}`]);
+//======Green text:======//
+navLinks.forEach(x => x.style.color = 'green');
```

Comment:
Nice loops! I would recommend making the variables more semantic (future-proofing your work) and putting both the green text and content logic into one forEach method:

```
navItems.forEach((link, index) => {
     link.textContent = siteContent['nav'][`nav-item-${index+1}`];
     link.style.color = 'green';
});
```

OR

```
for (let i=0; i < links.length; i++) {
  navItems[i].textContent = siteContent['nav'][`nav-item-${i+1}`];
  navItems[i].style.color = 'green';
};
```